test/integration/targets/ios_logging/tests/cli/basic.yaml

```diff
@@ -127,3 +127,34 @@
       - 'result.changed == true'
       - '"no logging console" in result.commands'
       - '"no logging buffered" in result.commands'
+
+- name: Configure logging trap
+  ios_logging:
+    dest: host
+    name: 172.16.0.1
+    facility: local6
+    level: warnings
+    trap: warnings
+    state: present
```

Comment:
@asand3r Please add `authorize: yes` to both the test cases.

Reply:
@trishnaguha Should I add two more tests or just correct present one?

Reply:
Just the current one that you added.